src/PKSim.Infrastructure/Serialization/ORM/Mappers/ProjectMetaDataToProjectMapper.cs

```diff
@@ -228,5 +227,10 @@ public void Visit(ImportPopulationMetaData importPopulation)
          _serializationManager.Deserialize(population.Settings, importPopulation.Properties.Data);
          _buildingBlock = population;
       }
+
+      public void Visit(ObserverSetMetaData observerSetMetaData)
```

Comment:
The new table in project database will be OBSERVER_SETS (I did not want to call the object ObserversMetaData otherwise the table would have been called OBSERVERS... and then it looks like a collection of single entities)